src/codemodder/codemods/remove_unused_imports.py

```diff
@@ -53,6 +66,38 @@ def filter_by_path_includes_or_excludes(self, pos_to_match):
             return any(match_line(pos_to_match, line) for line in self.line_include)
         return True
 
+    def _has_noqa_comment(self, node):
+        """
+        Check if the import has a #noqa comment attached to it
+        """
+        parent = self.get_metadata(ParentNodeProvider, node)
+        if parent and matchers.matches(parent, matchers.SimpleStatementLine()):
+            stmt = ensure_type(parent, cst.SimpleStatementLine)
+            pattern = re.compile(r"^#\s*noqa")
```

Comment:
Just a very small thing but this can be compiled once at the top of the module. It doesn't have a big performance impact in this case though.